src/compas_ghpython/artists/robotmodelartist.py

```diff
@@ -36,5 +43,13 @@ def create_geometry(self, geometry, name=None, color=None):
             mesh.FillHoles()
         return mesh
 
+    # should this not have a return value?!
     def draw(self):
+        """Draw the robot model.
+
+        Returns
+        -------
+        None
+
+        """
         self.draw_visual()
```

Comment:
yes, this is a bug. it should return `draw_visual()`  which is a generator of rhino meshes. 
```suggestion
    def draw(self):
        """Draw the visual meshes of the robot model.

        Returns
        -------
        generator[:rhino:`Rhino.Geometry.Mesh`]

        """
        return self.draw_visual()
```

And btw, the `draw_*` methods in the class `compas.artists.RobotModelArtist` are incorrectly documented as returning `None`, when most of them yield rhino meshes.

Reply:
and 2nd btw: is there a convention for how to write generators in the returns? or should it be changed to `sequence[]` instead?

Reply:
if it yields something it should be a generator, if it an ordered random access collection it should be a sequence. the proper type hint would be `Generator[...]` and `Sequence[...]`, but in my opinion lowercase is fine as well in docstrings

Reply:
👍  (yep, it's a generator)